### ISSUE TITLE
Publish Docker Image to DockerHub

### ISSUE THREAD

**Issue by zicklag** [int08h/roughenough#19]
Hello there,

It would be convenient to have the Docker image built and published to DockerHub. If you need a CI script to do it, I can create [Drone](https://drone.io) script for it.

**Comment by int08h** [comment]
@zicklag Agreed. If you can provide the Drone script I'll integrate it with the CI pipeline. 

Thanks for this, and for the PR!

**Comment by zicklag** [comment]
No problem. Thanks for the time server. :)
